docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md

```diff
@@ -436,7 +436,11 @@ espresso:
 ### `app`
 <p><small>| REQUIRED | STRING |</small></p>
 
-The path to the app. The default directory is `{project-root}/apps/filename.apk`, and the property supports expanded environment variables to designate the path, as shown in the following examples, or an already uploaded app reference. Supports \*.apk and \*.aab files.
+Either a local path, url, or storage identifier to the app under test. This property supports expanded environment variables.
```

Comment:
What is a storage identifier?

Reply:
The id of the file in the storage service. I guess its referred to as `file_id`? https://docs.saucelabs.com/mobile-apps/app-storage/#using-application-storage-with-automated-test-builds